i18n/en.yaml

```diff
@@ -1,7 +1,8 @@
 readMore:
   other: Read More
 readingTime:
-  other: '{{ .ReadingTime }} min read'
+  one: One min read
```

Comment:
I saw that `.Count` is a number instead of a work, would it be better if we use 1 instead of one?

Reply:
Surely we can